zk/tests/README.md

```diff
@@ -0,0 +1,76 @@
+# Quick Start
+
+The Zookeeper `ddev` environment has the option to have SSL enabled. SSL client and server test certificates can be found 
+within `tests/compose`. Sometimes, there might be issues with these certs and need to be reset. This guide goes through 
+the steps to generate new certs, since Zookeeper’s server does not use the widely-used `PEM` format for certs, and it 
+instead relies on the concept of `keystore` and `truststore`. A `keystore` contains the private key and the public cert 
+of a client/server, and the `truststore` contains the public certs of whoever the client/server trusts.
```

Comment:
```suggestion
You can enable SSL for your Zookeeper `ddev` environment. The SSL client and server test certificates can be found within `tests/compose`. There may sometimes be issues with these certs and they need to be reset. 

This guide covers the steps to generate new certs because Zookeeper’s server does not use the widely-used `PEM` format for certs—instead, it relies on `keystore` and `truststore` concepts. A `keystore` contains the private key and the public cert of a client/server. A `truststore` contains the public certs of whomever the client/server trusts.
```